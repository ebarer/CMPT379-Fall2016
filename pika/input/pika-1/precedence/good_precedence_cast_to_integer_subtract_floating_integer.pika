exec {

  print [ 7.0 | int ] - 44, _n_.
  print [ 3.3 | int ] - 6, _n_.
  print [ 1.8 | int ] - 46, _n_.
  print [ 7.5 | int ] - 96, _n_.
  print [ 4.5 | int ] - 56, _n_.
  print [ 9.4 | int ] - 97, _n_.
  print [ 6.9 | int ] - 69, _n_.
  print [ 5.9 | int ] - 1, _n_.
  print [ 6.6 | int ] - 62, _n_.
  print [ 1.8 | int ] - 74, _n_.
  print [ 6.2 | int ] - 96, _n_.
  print [ 9.7 | int ] - 18, _n_.

  print 57 - [ 1.3 | int ], _n_.
  print 82 - [ 7.9 | int ], _n_.
  print 49 - [ 1.5 | int ], _n_.
  print 41 - [ 3.6 | int ], _n_.
  print 18 - [ 8.0 | int ], _n_.
  print 36 - [ 7.1 | int ], _n_.
  print 47 - [ 4.1 | int ], _n_.
  print 40 - [ 1.6 | int ], _n_.
  print 68 - [ 4.4 | int ], _n_.
  print 23 - [ 6.5 | int ], _n_.
  print 72 - [ 4.1 | int ], _n_.
  print 99 - [ 2.8 | int ], _n_.

  print [ 6.5 | int ] - [ 9.2 | int ], _n_.
  print [ 3.9 | int ] - [ 4.4 | int ], _n_.
  print [ 6.8 | int ] - [ 7.6 | int ], _n_.
  print [ 5.8 | int ] - [ 8.9 | int ], _n_.
  print [ 4.0 | int ] - [ 5.8 | int ], _n_.
  print [ 1.9 | int ] - [ 1.5 | int ], _n_.
  print [ 4.3 | int ] - [ 4.0 | int ], _n_.
  print [ 1.6 | int ] - [ 7.7 | int ], _n_.
  print [ 2.6 | int ] - [ 6.1 | int ], _n_.
  print [ 1.1 | int ] - [ 1.0 | int ], _n_.
  print [ 3.2 | int ] - [ 8.0 | int ], _n_.
  print [ 4.6 | int ] - [ 7.1 | int ], _n_.

}

