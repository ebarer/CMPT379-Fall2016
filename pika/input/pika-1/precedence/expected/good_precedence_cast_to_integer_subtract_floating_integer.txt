-37
-3
-45
-89
-52
-88
-63
4
-56
-73
-90
-9
56
75
48
38
10
29
43
39
64
17
68
97
-3
-1
-1
-3
-1
0
0
-6
-4
0
-5
-3
